### PR TITLE
refactor(dsm): removes invision dsm

### LINKED ISSUE #238: Remove DSM Integration package
The DSM integration package is causing a few headaches:

1. Potential blocker for Storybook v6 upgrade
2. Not well maintained by DSM

It's been agreed between the package should be removed and an alternative method of displaying Angular components in the DSM should be adopted.

### PR DESCRIPTION
**Pull request by andy-polhill**
We have decided to remove Invision DSM and opt for a more manual process of linking examples. The DSM package had a very large dependency tree which sometimes led to issues. It also forced us into specific versions of dependences and at times held us back from upgrading Storybook.

fix #238

Storybook link: 
https://deploy-preview-258--legal-and-general-canopy.netlify.app/

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [x] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts
- [ ] I have [linked the new component](<(https://github.com/Legal-and-General/canopy/blob/master/docs/CONTRIBUTING.md#invision-dsm)>) to adobe DSM (if appropriate)

### CODE REVIEW

**Review by elenagarrone — APPROVED** [review]
Nice one. I was going to pick it up this morning but got pulled into something else 😅

File: .storybook/addons.js [hunk 1]
```diff
@@ -5,12 +5,9 @@ import '@storybook/addon-backgrounds/register';
...
 import '@storybook/addon-storysource/register';
-import { registerDsm } from '@invisionapp/dsm-storybook/register';
 import { addons } from '@storybook/addons';
 import canopyTheme from './canopy-theme';
 
 addons.setConfig({
   theme: canopyTheme,
 });
-
-registerDsm(process.env.STORYBOOK_DSM);
```
Comment (aaronBery) [inline .storybook/addons.js:16]: cool i guess this gets cleaned up manually after this branch merges :-)

**Review by owensgit — APPROVED** [review]

File: .storybook/addons.js (hunk 1 above)
Comment (aaronBery) [reply .storybook/addons.js:16]: as in removing the secret

**Review by aaronBery — APPROVED** [review]

File: .storybook/addons.js (hunk 1 above)
Comment (andy-polhill) [reply .storybook/addons.js:16]: we never put the secret in, it was only there for if you ran it locally.

**Pull request merged**